test/common/deployment_types.go

```diff
@@ -416,6 +537,30 @@ func TestStatefulSetsExpectedReplicas(t TestingTB, ctx *TestingContext) {
 				continue
 			}
 
+			if namespace.Name == RHSSOUserProductOperatorNamespace && product.Name == "keycloak" {
+				pods := &v13.PodList{}
+				selector, _ := labels.Parse("component=keycloak")
+				keycloakListOpts := []k8sclient.ListOption{
+					k8sclient.InNamespace(NamespacePrefix + "user-sso"),
```

Comment:
```suggestion
					k8sclient.InNamespace(RHSSOUserProductNamespace),
```
Assuming rebase with the latest changes from feature branch